src/app/components/elements/markup/portals/InlineDropZones.tsx

```diff
@@ -139,7 +139,7 @@ function InlineDropRegion({id, index, emptyWidth, emptyHeight, rootElement}: {id
 
     if (dropRegionContext && droppableTarget) {
         return ReactDOM.createPortal(
-            deviceSize === "xs" ? dropdownZone : draggableDropZone,
+            ((navigator.maxTouchPoints) || (deviceSize === "xs")) ? dropdownZone : draggableDropZone,
```

Comment:
We actually have a helper method for this! It uses `maxTouchPoints` in the same way, but has a second check to cover a few more bases.
```suggestion
            (isTouchDevice() || (deviceSize === "xs")) ? dropdownZone : draggableDropZone,
```
(you'll need to import it in the file rather than just accepting this suggestion)